CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+Thank you for your interest in making [FerretDB](https://github.com/FerretDB/FerretDB) better!
+
+## Finding something to work on
+
+We are interested in all contributions, big or small, in code or documentation.
+But unless you are fixing a very small issue like a typo,
+we kindly ask you first to [create an issue](https://github.com/FerretDB/github-actions/issues/new/choose),
+to leave a comment on an existing issue if you want to work on it,
+or to [join our Slack chat](https://github.com/FerretDB/FerretDB/README.md#community) and leave a message for us there.
+This way, you will get help from us and avoid wasted efforts if something can't be worked on right now
+or someone is already working on it.
+
+You can find a list of good issues for first-time contributors [there](https://github.com/FerretDB/github-actions/contribute).
+
+## Setting up the environment
+
+### Requirements
+
+**TODO**
```

Comment:
@noisersup do you need any help with this part?

Reply:
Not now I believe